static/translations/messages_en.json

```diff
@@ -1,4 +1,9 @@
 {
+  "Quote and reply": "Quote and reply",
+  "{username} [said]({link_to_message}):": "{username} [said]({link_to_message}):",
+  "[Quoting…]": "[Quoting…]",
+  "[Quoting ({invocationId})…]": "[Quoting ({invocationId})…]",
```

Comment:
Perhaps "({serialNumber})"? Might be a bit easier for people translating to understand :slightly_smiling_face: